src/__tests__/index.test.ts

```diff
@@ -300,6 +300,13 @@ const vectors: Array<TestVector> = [
       },
     ],
   },
+  {
+    name: 'VSYS',
+    coinType: 360,
+    passingVectors: [
+      { text: 'DsnBFk2BdqYP3WEmChpL7TSonhpxUAi8wiA', hex: '073fe8b089c48ba23c60c64c5226d47acfb26565e313934d5d73'},
```

Comment:
Is this actually a valid address? By looking into their explorer, it looks like all addresses start from `A` 

https://explorer.v.systems/transactions/GKW115Gg1Xvve88beVxHMRCZQMqXzuw95aaBKZ892g7Q

Reply:
Thanks. Resolved.